lib/generators/manageiq/plugin/templates/lib/%plugin_path%/engine.rb

```diff
@@ -11,5 +11,13 @@ def self.vmdb_plugin?
     def self.plugin_name
       _('<%= plugin_human_name %>')
     end
+
+    def self.init_loggers
```

Comment:
I was using this comment (and any other ones as well) for reference prior to giving my own thoughts:

https://github.com/ManageIQ/manageiq/pull/20950#discussion_r556837019

But one downside to this approach is that a plugin author can technically do whatever the heck they feel like in this section, and it could have nothing to do with `loggers`.  Highly doubt it will be a problem, but just something that came to mind as I was looking at this.

Reply:
Yes, we want them to do whatever they want.